### PR TITLE
Add accessibility review template

### PR DESCRIPTION
**Pull request by domoritz**
Please squash into one commit.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/accessibility.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    description: >
+      Please add comments on any assistive failures (even new ones not listed above). If it makes sense, add screenshots, and suggest improvements.
+  validations:
+    required: true
+
+- type: checkboxes
+  attributes:
+    label: Flexible
+    description: >
+      These checks ensure
```
Comment (mjskay) [inline .github/ISSUE_TEMPLATE/accessibility.yml]: @domoritz I think this is missing the end of the sentence...

**Comment by mjskay** [comment]
This is great, thanks so much! There's one minor comment about a missing sentence I left, after which I'm happy to merge this.

**Comment by domoritz** [comment]
Done. Please squash and merge. I might have some follow ups as I go through the review.

**Comment by frankelavsky** [comment]
Wow, this is really great work! Just hopping on to say (for the record) how excited I am to see this. I hope this can set a standard of practice that more journals follow.

**Comment by domoritz** [comment]
Thank you. I mostly copied the ideas from Chartability so thank you for assembling that resource.

**Comment by mjskay** [comment]
Awesome, thanks both!

**Pull request merged**